src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs

```diff
@@ -157,6 +176,48 @@ public Uri Endpoint
     /// </summary>
     public BatchExportProcessorOptions<Activity> BatchExportProcessorOptions { get; set; }
 
+#if NET6_0_OR_GREATER
+    /// <summary>
+    /// Gets or sets the trusted certificate to use when verifying a server's TLS credentials.
+    /// </summary>
+#else
+    /// <summary>
+    /// Gets or sets the trusted certificate to use when verifying a server's TLS credentials.
+    /// </summary>
+    /// <remarks>
+    /// This option does not affect <see cref="OtlpExportProtocol.HttpProtobuf"/> protocol.
```

Comment:
For now, maybe we should consider throwing a PlatformNotSupported exception. Ideally we avoid taking a dependency on something like BouncyCastle as you note https://github.com/open-telemetry/opentelemetry-dotnet/issues/2009#issuecomment-1657221586

Reply:
Yes, after much fruitless effort I've concluded that it's not worth implementing the support for older .NET, especially when much of the functionalities can be manually wired using `HttpClientFactory`.